### DISCUSSION TITLE
Feedback: UI

### DISCUSSION THREAD

**Discussion by Der-Flip** [OpenTracksApp/OpenTracks#1264] [General]
So, I'm a person who's not a fan of change, just so it's out pf the way: I kinda liked the old UI better :'D

Anyways, here's my two cents on it:
it's not bad but there's a few things that bug me. But first the good point:

I like the new prompt after stopping a track to edit TrackName and activity type. There have been a few times where I forgot to edit the activity type completely.
I also like how the stop button acts more like a "pause" button. Giving you the option to just pause the tracking through one click, rather than going through the menu to pause or to stop and then going through the menus to continue a track.

Let's get to the things I think could be improved:
continuing the point above: While I like the "pause screen" I can see it becoming annoying quite quickly. I think an option in the App settings to **_define the Stop Button's behaviour_** would be helpful. I'd prefer for the stop button to actually stop the tracking (the way it was before the ui update) and not pause it.
I would also suggest **i_ncreasing the time to hold the stop button_** ever so slightly, because at this point in time it feels very short (half a second or less? I would increase it to probably a second or longer).

Further a **_"delete track"_** button on the pause screen would be fairly nice but I also agree that this would be more a bonus than a necessary feature. Maybe it can be dependent on the track's length. Whenever a track is below 100 (or so) meters the delete / disregard button is being shown, anything above it won't be.

Let's continue with the menu bar.
The App Logo: As far as I noticed there is no function set to it. Yet it is being highlighted through colour. This doesn't make too much sense from an interaction focussed point of view. It would make sense it the menu items were to be hidden behind the logo (slide in / slide out menu) but not the way it is right now.

Icon size:
In my eyes the **_menu icons_** are **_too small_**. This may be due to the **_menu bar_** being a little **_too small_** as well.

Scrollable content hint:
In order make the user understand they can scroll, it is common (and good) practice to have the lower two thirds / lower half of the last element cut off. This is (at least on my device, redmi9C) not the case at the moment, instead the last list element when scrolled to the very top of the screen. Despite knowing I should be able to scroll through, my initial thought seeing my list was 'can I still scroll through?'. Basically: don't allow a whole number list items to fit on the screen at any point. 

Increasing the height of the menu bar could fix this issue easily and would also give you more room for larger icons.

Alternatively one could get rid of the menu bar entirely, have the app logo as big as the record button and have the menu items slide out (quick and dirty sketch: 
<img src="https://user-images.githubusercontent.com/18213041/171026110-9f05ed1f-e82d-4906-b09a-3358c17aecea.png" width="75%" align="center" />
)

lastly: from a colour design point of view I'd suggest making the menu bar the same grey as the grey seen on the app's splash screen.

I know it's a lot of things and if I wasn't so bad at coding I most certainly would help but trust me, you don't want me touching your code in the slightest.

**Comment by dennisguse** [comment]
@Der-Flip all valid things :)
Just create issues for each of them - otherwise we have might a very confusing discussion :sunglasses: 

PS/ for the menu bar only one issue ;)

PPS/ nice drawing :D

**Reply by Der-Flip** [reply]
it is done :'D
I hope it's alright how I did it.

please keep in mind, that despite my phrasing at some points, those are merely some suggestions / opinions, and not entirely to be seen as requests. Although it was part of my course of studies, I am not a professional UI / UIX designer. These are just some things that caught my (personal) eye. It's perfectly fine to disagree :)

**Reply by dennisguse** [reply]
No worries :sunglasses:
